geolocator_apple/ios/Classes/Handlers/GeolocationHandler.h

```diff
@@ -20,6 +20,8 @@ typedef void (^GeolocatorResult)(CLLocation *_Nullable location);
 
 - (void)startListeningWithDesiredAccuracy:(CLLocationAccuracy)desiredAccuracy
                            distanceFilter:(CLLocationDistance)distanceFilter
+pauseLocationUpdatesAutomatically:(NSNumber* _Nullable)pauseLocationUpdatesAutomatically
```

Comment:
Don't accept the `NSNumber` type here if we know we need a boolean (this will force calling code to deal with conversions):
```suggestion
pauseLocationUpdatesAutomatically:(BOOL)pauseLocationUpdatesAutomatically
```